api/pacman-api-admin/src/main/java/com/tmobile/pacman/api/admin/repository/model/AccountDetails.java

```diff
@@ -80,4 +82,16 @@ public String getPlatform() {
     public void setPlatform(String platform) {
         this.platform = platform;
     }
+    public AccountDetails() {
+    }
+    public AccountDetails(String accountId, String accountName, String assets, String violations, String accountStatus, String platform, String createdBy, Timestamp createdTime) {
```

Comment:
Consider refactoring the function parameters into a class to encapsulate the essential properties and enhance code readability.

Reply:
Hi paul, i can create a AccountDetailsDTO. But it will be used only in one place and not in multiple places. We shall create in future if it is needed in multiple places. For now, i don't want to add a new file to keep the code lean.